presto-main/src/main/java/com/facebook/presto/execution/CreateViewTask.java

```diff
@@ -82,7 +82,7 @@ public String explain(CreateView statement, List<Expression> parameters)
 
         accessControl.checkCanCreateView(session.getRequiredTransactionId(), session.getIdentity(), name);
 
-        String sql = getFormattedSql(statement.getQuery(), sqlParser, Optional.of(parameters));
+        String sql = SqlFormatter.formatSql(statement.getQuery(), Optional.of(parameters));
```

Comment:
We definitely want round-trip validation here. The purpose is to catch bugs in the formatter that silently would create an invalid view (or worse, one that works but has the wrong behavior).

The bug needs to be fixed in the formatter.